views/xibo-rdm-connector-form-javascript.twig

```diff
@@ -0,0 +1,501 @@
+                  deviceTable.ajax.reload();
+                },
+                error: function(error) {
+                  toastr.error('Failed to link device. Please try again.');
+                }
+              });
+
+              resetSelection();
+
+              // Revert modal z-index changes
+              adjustModalZIndex(1050)
+            }
+          }
+        }
+      }).attr('id', Date.now());
+    }
+
+    function resetSelection() {
+      selectedDisplay.id = '';
+      selectedDisplay.name = '';
+      selectedDevice.id = '';
+      selectedDevice.name = '';
+    }
+
+    // Adjust z-index for stacked modals
+    function adjustModalZIndex(zIndex) {
+      $('.rdm-modal').css('z-index', zIndex);
+    }
+  };
+
+</script>
+<script type="text/x-handlebars-template" id="connector-rdm-display">
+  <div class="XiboGrid" id="{{ gridId }}" data-grid-name="connector-rdm-display">
+      <!-- Card Header -->
+      <div class="card-header">
+          <ul class="nav nav-tabs card-header-tabs" role="tablist">
+              <li class="nav-item">
+                  <a class="nav-link active" id="unlinked-tab" data-toggle="tab" href="#unlinkedTab" role="tab"
+                     aria-controls="unlinkedTab" aria-selected="true">Unlinked</a>
+              </li>
+              <li class="nav-item">
+                  <a class="nav-link" id="linked-tab" data-toggle="tab" href="#linkedTab" role="tab"
+                     aria-controls="linkedTab" aria-selected="false">Linked</a>
+              </li>
+          </ul>
+      </div>
+
+      <!-- Card Body -->
+      <div class="card-body">
+          <div class="tab-content">
+
+              <!-- UNLINKED TAB-->
+              <div class="tab-pane active" id="unlinkedTab" role="tabpanel" aria-labelledby="unlinked-tab">
+                  <div class="row">
+                      <div class="XiboFilterCustom pb-0 mb-0 col-md-6 pr-lg-4">
+                          <div class="FilterDiv" id="connector-display-unlinked-table">
+                              <h5 class="m-3">{% trans "CMS Displays" %}</h5>
+                              <form id="connector-display-filter" class="form-inline m-3">
+                                  {% set title %}{% trans "Display Name" %}{% endset %}
+                                  {{ inline.input("display", title) }}
+                                  {% set title %}{% trans "Type" %}{% endset %}
+                                  {% set title %}{% trans "Type" %}{% endset %}
+                                  {% set options = [
+                                      { optionid: "", option: "All" },
+                                      { optionid: "Sony Bravia", option: "Sony Bravia" },
+                                      { optionid: "DSDevices", option: "DSDevices" },
+                                      { optionid: "Windows", option: "Windows" },
+                                      { optionid: "Chrome OS", option: "Chrome OS" },
```

Comment:
It should be `ChromeOS` please

I'm not entirely sure if we should list ChromeOS/Windows as only Sony Bravia and to some extent DSDevices will be supported for now - but that can be changed later on